runck

```diff
@@ -77,6 +87,8 @@ launch() {
     chmod 0755 "${tmpfile}"
   fi
 
+  echo ""
```

Comment:
i.e. we should add a `1>&2` when we echo CK=... and the output of sha256sum check